src/IPHlpApi/IPHlpApi+TcpTableType.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="TcpTableType"/> nested type.
+    /// </content>
+    public static partial class IPHlpApi
+    {
+        /// <summary>
+        /// The <see cref="TcpTableType"/> enumeration defines the set of values used to indicate the type
+        /// of table returned by calls to <see cref="IPHlpApi.GetExtendedTcpTable"/>.
+        /// </summary>
+        /// <seealso href="http://msdn2.microsoft.com/en-us/library/aa366386.aspx"/>
+        public enum TcpTableType
+        {
+            /// <summary>
+            /// A MIB_TCPTABLE table that contains all listening (receiving only) TCP endpoints on
+            /// the local computer is returned to the caller.
+            /// </summary>
+            BasicListener,
```

Comment:
Per our [contribution doc](https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#naming), please use the exact names as defined in docs and the .h file. For example this should be `TCP_TABLE_BASIC_LISTENER`.